config/config.js

```diff
@@ -4,6 +4,19 @@ var token = require("./slack_token.js").token;
 var private_key = require("./service_account_key.js").key;
 
 var config = {
+    // Unfortunately this all has to be at the top level due to sequelize-cli
```

Comment:
We can at least move it to the bottom of the config so it's not the most prominent part of the config.